src/toil/jobStores/fileJobStore.py

```diff
@@ -90,6 +90,7 @@ def resume(self):
         if not os.path.isdir(self.jobStoreDir):
             raise NoSuchJobStoreException(self.jobStoreDir)
         super(FileJobStore, self).resume()
+        self.linkImports = self.config.linkImports
```

Comment:
I don't think it makes sense to tie this to `linkImports`. If we can, we should *always* symlink or hardlink from the file store, unless the caller doing the read explicitly needs a mutable copy or a non-symlink file.

If we want to permit links by default, with the current symlink-only-if-allowed-explicityl semantics, we could just touch [here](https://github.com/DataBiosphere/toil/blob/8057b47f3841f177e040def8ef1b2f1968f0cd94/src/toil/cwl/cwltoil.py#L300) and [here](https://github.com/DataBiosphere/toil/blob/8057b47f3841f177e040def8ef1b2f1968f0cd94/src/toil/cwl/cwltoil.py#L309) where Toil's CWL runner reads from the `FileStore`, and have it pass `symlink=True`.

But I think really what we should do is just change the default for `symlink` to True, and make workflows that can't use symlinks demand non-symlinks or mutable copies.

Reply:
I agree with this.  This seems like adding a `dev` lever to force symlinks in special cases, but it really should be implemented as a broader design decision.  I think defaulting to using symlinks as a default should be tested.